package.json

```diff
@@ -94,7 +94,7 @@
     "@babel/preset-typescript": "^7.7.4",
     "@babel/register": "^7.7.4",
     "@babel/runtime": "^7.7.6",
-    "@types/react": "^16.9.17",
+    "@types/react": "16.9",
```

Comment:
Why is this no longer a caret dependency (also on https://github.com/matrix-org/matrix-react-sdk/pull/4203)?

Reply:
because yarn is awful for two reasons: it doesn't default to caret when installing things, and we should be pinning our types to our react version. `16.9` still gets resolved the same as `16.9.x`, which is compatible with carets.

Reply:
So `16.9`, `^16.9` `^16.9.0` all get resolved differently? Also, if this is the correct treatment for `@types/react`, should `@types/react-dom` below also be updated?

Reply:
![image](https://user-images.githubusercontent.com/2403652/77667829-9bdf0900-6f7a-11ea-9bcb-7084567189b9.png)
![image](https://user-images.githubusercontent.com/2403652/77667846-a0a3bd00-6f7a-11ea-80ab-6bfa2d940102.png)
![image](https://user-images.githubusercontent.com/2403652/77667871-a6999e00-6f7a-11ea-9495-3691a3d2dcd3.png)

src: https://semver.npmjs.com/